docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-product-offer-management-feature-integration.md

```diff
@@ -114,10 +115,36 @@ class DashboardMerchantPortalGuiDependencyProvider extends SprykerDashboardMerch
 }
 ```
 
+**src/Pyz/Zed/SalesMerchantPortalGui/SalesMerchantPortalGuiDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\SalesMerchantPortalGui;
+
+use Spryker\Zed\ProductOfferMerchantPortalGui\Communication\Plugin\SalesMerchantPortalGui\ProductOfferMerchantOrderItemTableExpanderPlugin;
+use Spryker\Zed\SalesMerchantPortalGui\SalesMerchantPortalGuiDependencyProvider as SprykerSalesMerchantPortalGuiDependencyProvider;
+
+class SalesMerchantPortalGuiDependencyProvider extends SprykerSalesMerchantPortalGuiDependencyProvider
+{
+    /**
+     * @return \Spryker\Zed\SalesMerchantPortalGuiExtension\Dependency\Plugin\MerchantOrderItemTableExpanderPluginInterface[]
+     */
+    protected function getMerchantOrderItemTableExpanderPlugins(): array
+    {
+        return [
+            new ProductOfferMerchantOrderItemTableExpanderPlugin(),
+        ];
+    }
+}
+```
+
```

Comment:
not needed, presented in Marketplace Merchant Portal Product Offer Management + Merchant Portal Order Management feature integration